### DISCUSSION TITLE
new functionalities to force strict usage of rules files

### DISCUSSION THREAD

**Discussion by sched75** [cline/cline#663] [Feature Requests]
I have designed rules files very extensive about the way some languages must be used for programming but even if ask cline to use them, it seems there is too many rules to follow them all. 

so I'd like we find soon in cline an action @rules like aider just introduced it. @steps for defining programming steps to follow would be great too.

**Comment by d0lwl0b** [comment]
We can jointly implement a "mandatory" standard for LLMs.

see: #5912
